internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -213,6 +213,12 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, aws aw
 		return err
 	}
 
+	logger.Info("Adding https://kiwigrid.github.io Helm repo.")
+	addHelmRepo("kiwigrid", "https://kiwigrid.github.io", logger)
```

Comment:
We are ignoring the returned error here. We should check it.

Reply:
Thanks @gabrieljackson fixed :)